### ISSUE TITLE
Irregular and confusing defaultRepresentation options

### ISSUE THREAD

**Issue by tcharlss** [Simonwep/pickr#83]
#### Do you want to request a **feature** or report a **bug**? 

It seems like a bug.

#### What is the current behavior?

I'm using HEXA as the `defaultRepresentation` parameter, and an hexadecimal value as the default value (let's say `#42445A`).
In that case, the input component remains empty, even when I press the save button.
As soon as I enter manually a value in the input, everything is back on track.

There's no such problem with any other format (hsla, cmyk, etc.).

By the way, is the  `defaultRepresentation` parameter mandatory? Is it auto-detected based on the default value if I omit that parameter?

![pickr_bug](https://user-images.githubusercontent.com/9882128/56309657-44b99d00-614a-11e9-83a2-e709fc788114.png)

#### If the current behavior is a bug, please provide the steps to reproduce and create a [JSFiddle](https://jsfiddle.net/).

Here is a Fiddle with the exact setup I'm using:
https://jsfiddle.net/tcharlss/6mfx029g/4/

#### Your environment:
```
Pickr-version (see Pickr.version): 4.9+ (master branch)
Used Pickr-bundle (es5 or normal one): normal
Browser-version: Firefox 66.0.3
Operating-system: Linux ubuntu 18.10
```

**Comment by tcharlss** [comment]
> As soon as I enter manually a value in the input, everything is back on track.

I stand corrected, in that case it updates the palette, but not the other way around.

**Comment by Simonwep** [comment]
`HEXA` is not a valid `defaultRepresentation`, see [options](https://github.com/Simonwep/pickr#options). And yes, if you use, for example, a color in `rgba` format *and* the `rgba` representation button is enabled it get's automatically selected :)

Use `HEX` if you want to use hexadecimal values. I'll add `HEXA` support until the next release 😃

**Comment by Simonwep** [comment]
Actually this doesn't need to get implemented because all kind of non-opacity formats like `RGB`, `HSL` gets threaded as `RGB`. I'll consider a refactoring of this config property.

**Comment by Simonwep** [comment]
Done in 9f716f99f042b73f1de128289e30e87e7c39b915, available in 0.4.10
